packages/db-kit/src/cli/components/codec/Constructor.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Box, Text, Newline } from "ink";
+
+import type { Transaction, TransactionReceipt } from "web3-core";
+import { ConstructorDecoding } from "@truffle/codec";
+
+import { DefinitionList } from "@truffle/db-kit/cli/components/DefinitionList";
```

Comment:
Yeah I dunno why, relative imports make me cringe. This follows the pattern I established in @truffle/db for importing outside a semantic boundary. This works because I'm using tsonfig's `paths` compiler setting along with some other wizardry that I'll spare the details of unless you're curious.

Reply:
Thats fine with me